src/components/ConfirmationBox/ConfirmationBox.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ConfirmationBox from './ConfirmationBox';
+
+configure({adapter: new Adapter()});
+
+it('calls the correct functions when clicked', () => {
```

Comment:
I would probably split this up, in most cases it is best to only have one expect per test. If yes fails here but no passes you will have to dig into it to find which is failing.
"calls onYes function when yes is clicked" or something

Reply:
done